ruby-challenges.rb

```diff
@@ -19,11 +28,29 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+# psuedocode:
+# 1. Create a method
+# 2. Take in an array of numbers
+# 3. Return the sum of the numbers
+
+
+def sum_nums(array)
+  array.reduce(:+)
+end
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
+# Psuedocode:
+# Create a class
+# Initialize with a model, wheels, and current_speed
+# Create a bike_info method that returns a sentence with all the data from the bike object
+
+
+def bike_info(model, wheels, current_speed)
```

Comment:
Make sure to read closely what the prompt is asking.  In this case, you should be setting up a class that has this method inside of it.  I would recommend taking a look at the [Ruby classes](https://github.com/learn-academy-2022-delta/Syllabus/blob/main/ruby/classes-objects.md) section in the syllabus.